chips/stm32f412g/README.md

```diff
@@ -0,0 +1,5 @@
+# ST Micro stm32f412g MCU
+
+Builds upon the `stm32f4xx` crate and includes hardware setup that is specific to this version of
+the stm32f4xx series. Boards that include an stm32f429zi MCU should use this crate as a dependency
```

Comment:
> Boards that include an **stm32f429zi MCU**

Quick sanity check: I'm afraid the answer is going to be that STM just has a confusing naming scheme, but the name / title of this file is `stm32f412g`. Is this all correct as-written?

Reply:
It is based on the same stm32f4 family, but the stm32f429zi line is wrong. Fixing it.